docs/src/statmodels.md

```diff
@@ -30,7 +31,6 @@ rss
 score
 stderror
 vcov
-weights(::StatisticalModel)
```

Comment:
Why not keep this?

Reply:
There is no longer a specific definition in StatsAPI - its just `function weights end`.

`weight` cant stay here either because it's already included in weights.md, so you get a duplicate entry error

Reply:
Ah, indeed. That's annoying. I guess the only solution is to include just `weights` here, but make the signatures in weights.md more specific.

Reply:
Ok I've reshuffled this so it can still be in both, with the specific signature in weights.md